src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -360,6 +380,7 @@ export class InputTimePicker implements LabelableComponent, FormComponent {
           aria-haspopup="dialog"
           aria-label={this.name}
           aria-owns={popoverId}
+          dir={this.calciteInputElDir}
```

Comment:
If we do need to do direction styling by language, could you instead try using the lang psuedo selector?

https://developer.mozilla.org/en-US/docs/Web/CSS/:lang


```scss
div:lang(ar) {
  direction: rtl;
}
```

If this works, we wouldn't need to use getElementDir or wait for another render.